server/src/controllers/Chapter/resolver.ts

```diff
@@ -68,6 +68,7 @@ export class ChapterResolver {
               },
             },
             user: { include: { instance_role: true } },
+            chapter: true,
```

Comment:
```suggestion
```
This should not have changed.

Reply:
True, I thought the revert removed them. 👍